### PR TITLE
Update Daml Shell docs with filter/export functionality

### PR DESCRIPTION
**Pull request by iggy-da**
- Regenerate gifs with new Daml Shell version
- Update docs with export/csv functionality

### CODE REVIEW

File: docs/2.9.0/docs/tools/daml-shell/index.rst [hunk 1]
```diff
@@ -3,30 +3,36 @@
...
+-  Find all events related to a transaction ID. A transaction is
+   displayed as a series of ledger events (creates, archives, and
+   exercises).
+-  Auto-completion for identifiers such as contract IDs, fully qualified
+   names, and package IDs.
+-  List active, inactive, or all contracts for a template using a
+   template FQN.
+-  Apply queries and filters to commands to manage the output.
+-  Use the ``compare-contracts`` command to highlight the delta between
+   two contract IDs for the same template.
```
Comment (iggy-da) [inline docs/2.9.0/docs/tools/daml-shell/index.rst:29]: The formatting changes are due to upstream conversion from markdown to rst

File: docs/2.9.0/docs/tools/daml-shell/index.rst [hunk 2]
```diff
@@ -3,30 +3,36 @@
...
 
 To bound a search, specify a minimum and maximum offset value.
 
-Daml Shell leverages PQS as its datastore and connects using JDBC. For more information, see :ref:`pqs-connect-header`.
-
+Daml Shell leverages PQS as its datastore and connects using JDBC. For
+more information, see
+:ref:`configuration`
```
Comment (iggy-da) [inline docs/2.9.0/docs/tools/daml-shell/index.rst:35]: I've changed this link to point to the Daml Shell connection configuration. The flags follow the same naming scheme, but are different (e.g. some values have a prefix in PQS while they do not have one in Daml Shell)

File: docs/2.9.0/docs/tools/daml-shell/index.rst [hunk 3]
```diff
@@ -513,6 +539,28 @@ To look up individual exercises, use the event ID:
...
+
+You can then write this output to a file by piping it through the
+``export`` filter:
+
+::
+
+   > active PingPong:Ping | csv | export ~/my_pings.csv
+
+The ``export`` filter will write any command output to the specified
+file. You can use it without the ``csv`` filter.
```
Comment (carrielaben-da) [inline docs/2.9.0/docs/tools/daml-shell/index.rst:560]:
```suggestion
The ``export`` filter writes any command output to the specified
file. You can use it without the ``csv`` filter.
```

**Review by carrielaben-da — APPROVED** [review]
Approved with minor changes.

**Pull request merged**